x509-identity-mgmt/ejbca/bin/before-shutdown.sh

```diff
@@ -1,18 +1,5 @@
 #!/bin/bash
 
-##################################################################
-#                                                                #
-# Copyright (c) 2018-2019 PrimeKey Solutions AB.                 #
```

Comment:
Is this file written by PrimeKey and modified by us? In affirmative case, I think we should add a Notice file saying this and depending on the license used by PrimeKey this specific file must continue be licensed under its original license.